### DISCUSSION TITLE
Circular microstrip patch antenna design

### DISCUSSION THREAD

**Discussion by ssmulla** [LubomirJagos/FreeCAD-OpenEMS-Export#24] [General]
Plz explain how to create circular metal patch. I have created using AddCylinder class but not getting results. I am using python interface.

**Comment by LubomirJagos** [comment]
that python interface is really experimental, like in sense I wouldn't use it at all, month ago I implemented script generate for it, but since then I added lot of functionality, octave interface is reference one, that one is which I test simulation against and python one is waiting for implementation, I will try to add things into it, but for now it's missing more than 50% what it should be
